tokio-trace-fmt/src/lib.rs

```diff
@@ -288,14 +306,12 @@ impl<N, E, F> Builder<N, E, F> {
 
     /// Sets the subscriber being built to use the default full span formatter.
     // TODO: this should probably just become the default.
-    pub fn full(
-        self,
-    ) -> Builder<N, fn(&span::Context<N>, &mut fmt::Write, &Event) -> fmt::Result, F>
+    pub fn full(self) -> Builder<N, default::Standard, F>
```

Comment:
As I mentioned above, this should _probably_ be made the default, and we should add a `short` configuration or something.

Reply:
I named it `compact` after the similar `slog` feature.